src/index.scss

```diff
@@ -1,2 +1,3 @@
 @import "~normalize-scss/sass/normalize/import-now";
 @import "~bootstrap/scss/bootstrap";
+@import "/style.scss";
```

Comment:
Nitpick, but `"./style"` is a little more specific and concise.

Reply:
fixed